apis/management.cattle.io/v3/authn_types.go

```diff
@@ -196,12 +196,14 @@ type ActiveDirectoryConfig struct {
 	UserDisabledBitMask          int64    `json:"userDisabledBitMask,omitempty"         norman:"default=2"`
 	UserSearchBase               string   `json:"userSearchBase,omitempty"              norman:"required"`
 	UserSearchAttribute          string   `json:"userSearchAttribute,omitempty"         norman:"default=sAMAccountName|sn|givenName,required"`
+	UserSearchQuery              string   `json:"userSearchQuery,omitempty"`
```

Comment:
I think the name needs changed to clarify the purpose (that its an additional filter, not a replacement).
I've asked for @vincent99 but until we hear back, lets go with `additionalUserSearchFilter`